docs/developers/add-external-chart.rst

```diff
@@ -32,38 +32,47 @@ No published Helm chart
 If the application has an existing, maintained Helm chart, but it's not published in a Helm repository, this is also a red flag, albeit a lesser one.
 In exceptional circumstances we can import such an external Helm chart into the `charts repository <https://github.com/lsst-sqre/charts/>`__, but we would prefer not to do this since keeping it up-to-date with upstream changes is very awkward.
 
-.. _external-chart-config:
+Create a parent Helm chart
+==========================
 
-Configure the external chart
-============================
+If you are deploying a new service (and not, for instance, adding an external chart to an existing Phalanx application), start by creating a new chart.
+For charts designed primarily to deploy an external chart, normally you want to use the empty Helm starter:
 
-Configuration mostly involves carefully reading the documentation of the upstream Helm chart and building a ``values.yaml`` file that configures it appropriately.
-You may also need to add additional resources not created by the upstream Helm chart, particularly ``VaultSecret`` objects to create any secrets that it needs.
-See :doc:`add-a-onepassword-secret` for more about secrets.
+.. code-block:: shell
+
+   $ cd applications
+   $ helm create -p $(pwd)/../starters/empty <application>
```

Comment:
If you want to include prompt characters to imply an interactive shell, use the `prompt` directive instead so that the prompt characters aren't included when the user copies the code cell.

```
.. prompt:: bash

   cd applications
   helm create -p $(pwd)/../starters/empty <application>
```